README.rst

```diff
@@ -0,0 +1,99 @@
+==================
+nethserver-cockpit
+==================
+
+NethServer Cockpit UI and new Server Manager.
+
+The document describe how to prepare an environemnt to develop
+
+Prepare the environment
+=======================
+
+- Install developer tools: npm and git
+
+  On Fedora: ::
+
+    dnf install npm git
+
+  On CentOS: ::
+
+    yum install npm git
+
+- Clone the repository: ::
+
+    git clone git@github.com:NethServer/nethserver-cockpit
+
+- Install grunt globally: ::
+
+    npm install -g grunt-cli bower generator-angular yo
+
+
+  You can ignore warnings like this: ::
+
+    WARN generator-angular@0.16.0 requires a peer of generator-karma@>=0.9.0 but none was installed.
+
+
+- Enter inside the cloned repository and install all dependencies: ::
+
+    cd nethserver-cockpit/
+
+    cd api/system/ && npm install
+    cd ..
+    cd ui/ && npm install && bower install
+
+Build API and UI
+================
+
+Enter API directory and build using grunt: ::
+
+  cd api/system/ && grunt build
+
+
+Enter UI directory and build using grunt: ::
+
+  cd ui/ && grunt build && bower install
```

Comment:
I think is enough `cd ui/ && grunt buil` instead of `cd ui/ && grunt build && bower install`, because the `bower_components` directory is already filled in the previous step.